x-pack/plugins/enterprise_search/server/lib/indices/fetch_index.ts

```diff
@@ -35,14 +36,20 @@ export const fetchIndex = async (
     };
   }
 
-  const crawlerResult = await client.asCurrentUser.search<Crawler>({
-    index: '.ent-search-actastic-crawler2_configurations',
-    query: { term: { index_name: index } },
-  });
-  const crawler = crawlerResult.hits.hits[0]?._source;
-
-  if (crawler) {
-    return { ...indexResult, crawler };
+  try {
```

Comment:
There is another change on this file that is open. We want to fail silently on the other case, we probably should talk what is the expected outcome. 
https://github.com/elastic/kibana/pull/137108/files#diff-e41c77f9227200be5fa4bf7fbca56524a23cc1c210f78e4ef35aa806f7ab4b94 

cc @sphilipse

Reply:
Oh yeah! I did this because otherwise I couldn't navigate to the index overview page.

For whatever reason, I don't have the connectors index, so the 404 from *that* was erroring out the page and sending me back to the indices listing.

I'll take a look at the merge conflict.

Reply:
Oh I see, this error is now caught:
https://github.com/elastic/kibana/blob/0523cd201916db1039406e87975fabcfe9d0634e/x-pack/plugins/enterprise_search/server/lib/crawler/fetch_crawlers.ts#L46-L48

I'll back this commit out 👍